lib/core/Schema.js

```diff
@@ -58,7 +58,7 @@ const MutatingWorkflowContextSchema = {
 const ReadOnlyWorkflowContextSchema = {
   type: 'object',
   properties: {
-    iamUser: {
+    ramUser: {
```

Comment:
I think it's supposed to be `iamUser` since the product is named IAM on AWS?

Reply:
Yes, my bad. I included the wrong commit from #1